app/database/migrations/20211108144203_create_decrypted_amounts_table.ts

```diff
@@ -0,0 +1,36 @@
+import { Knex } from 'knex';
+import {
+    decryptedAmountsTable,
+    accountsTable,
+    transactionTable,
+} from '~/constants/databaseNames.json';
+
+export async function up(knex: Knex): Promise<void> {
```

Comment:
Should this copy the already decrypted amounts from the transactions table into the decryptedAmountsTable?

Reply:
That's a good question. The migration can become quite slow if we do not just drop everything. I can try to profile it a little bit before we decide.

Reply:
We have decided that it will be okay for people to decrypt again after the upgrade. There will be few users that have encrypted transfers at the point of the release, so we will skip it here.